include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -16113,9 +16125,9 @@
           "version" : "None"
         },
         {
-          "enumerant" : "LongConstantCompositeINTEL",
+          "enumerant" : "LongCompositesINTEL",
           "value" : 6089,
-          "extensions" : [ "SPV_INTEL_long_constant_composite" ],
+          "extensions" : [ "SPV_INTEL_long_composites" ],
```

Comment:
Why did the names change? This looks like a backwards incompatibility. It is ok to add an extra entry for the same token number if they alias.